docs/app/partials/docs-demo.tmpl.html

```diff
@@ -22,6 +22,11 @@
               Source
             </div>
           </md-button>
+          <md-button ng-click="demoCtrl.editOnCodepen()"
```

Comment:
On small screens, the buttons don't have enough room to sit side-by-side. We have a few options:
 * Reduce to the icon only, although the icons are ambiguous.
 * Drop buttons to the next line, under "Basic Usage".

![Button demo simulated on Apple iPhone 5](https://cloud.githubusercontent.com/assets/1045233/7503698/8b2111be-f3f9-11e4-820b-5947957a0450.png)